### PR TITLE
FEATURE: CTRL-SHIFT-ENTER and SHIFT-Click do not scroll on post 

### PR DESCRIPTION
**Pull request by SamSaffron**
- FEATURE: CTRL-SHIFT-ENTER and SHIFT-Click do not scroll on post
- Address feedback from review
- Only forward events when consumers opt in

### CODE REVIEW

**Comment by SamSaffron** [comment]
@eviltrout  / @jjaffeux  this should address the instability here. You now need to opt in to event forwarding. I am liking this design, feels a lot less fragile.

File: app/assets/javascripts/discourse/app/controllers/composer.js [hunk 1]
```diff
@@ -546,8 +546,8 @@ export default Controller.extend({
       this.cancelComposer(differentDraftContext);
     },
 
-    save() {
-      this.save();
+    save(ignore, event) {
```
Comment (riking) [inline app/assets/javascripts/discourse/app/controllers/composer.js:549]: Linters prefer `_ignored` here, or `_force`

**Review by eviltrout — APPROVED** [review]
I like it!

File: app/assets/javascripts/discourse/app/controllers/composer.js (hunk 1 above)
Comment (SamSaffron) [reply app/assets/javascripts/discourse/app/controllers/composer.js:549]: our linter did not complain though? not against changing it but seems odd that there were no complaints.

**Pull request merged**